prediction/src/tests/test_endpoints.py

```diff
@@ -18,7 +18,7 @@ def skip_slow_test():
     """
     Skip the wrapped test function unless the environment variable RUN_SLOW_TESTS is set.
     """
-    return os.environ.get('RUN_SLOW_TESTS') != '1'
+    return not os.environ.get('RUN_SLOW_TESTS')
```

Comment:
Ok... you're gonna hate me for this... I think I spoke too soon. I'm use to python configs which get parsed to python objects. OS env values are returned as strings so you were right!! Apologies. :) The remaining question though becomes how to handle non `'1'` values. Since `local.yml` is a yaml file, I think some people would expect to be able to enter `True` as well. Borrowing from what I [read here](https://stackoverflow.com/a/43087058/408556) something like this may work.

```python 
value = os.environ.get('RUN_SLOW_TESTS')
return value.lower() not in {'1', 'true'}
```

Thoughts??

Reply:
When I see that there is a `RUN_SLOW_TESTS=0` in the YAML file and I'd like to run the slow tests, I'd change it to `RUN_SLOW_TESTS=1`. On the other hand, adding one line of code shouldn't be labeled "over-engineered" IMO :)